pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,91 @@
+{stdenv, mkDerivation,fetchFromGitHub,
+cmake, gcc, pkgconfig,glew,opencv3,libusb,libopus,
+boost,qtbase,qtwebsockets,qtmultimedia,wrapQtAppsHook,
+fftw,fftwFloat,fftwLongDouble,fftwSinglePrec,
+limesuite,
+ocl-icd,
+pulseaudioSupport ? true,libpulseaudio,libav,libiio
+}:
+
+assert pulseaudioSupport -> libpulseaudio != null;
+
+let
+  qtlibs = [
+    qtbase qtwebsockets qtmultimedia wrapQtAppsHook
+  ];
+  fftwlibs = [
+    fftw fftwFloat fftwLongDouble fftwSinglePrec
```

Comment:
You should only need one of them (`fftwFloat` would be my guess).

Reply:
with just fftw it didn't worked and i added them all out of lazyness i could try removing the others